content/blog/partnership-with-junior-guru.md

```diff
@@ -0,0 +1,29 @@
++++
+title = "Partnership With junior.guru"
+description = "This next episode of our interview series includes a chat with Daria Grudzien - one of CyberMagnolia's Co-Founders."
```

Comment:
This is a leftover from the previous article I guess :) 
We can use something from the article, for example "We proudly announce our first partnership with an inspiring tech community junior.guru created by Honza Javorek."

Reply:
Good catch, thank you! <3